testforwebhook.test

```diff
@@ -1 +0,0 @@
-test
```

Comment:
What is this for?

Reply:
That was a test when configuring webhooks for discord.

Reply:
Should be deleted (not emptied).